assets/dev/scss/editor/navigator/navigator.scss

```diff
@@ -113,9 +110,10 @@ body {
 		&__promotion {
 			font-size: 12px;
 			padding: 4px 17px;
-			line-height: 22px;
 			font-family: 'Roboto', sans-serif;
 			color: var(--e-a-bg-logo);
+			min-height: 30px;
```

Comment:
Maybe use vars ?
@louiswol94 wdyt ?

Reply:
Yeah, not sure if `min-height` is needed.